lib/utils/color-utils.ts

```diff
@@ -24,3 +24,22 @@ export function stringToHSLAColor({
 
   return `hsla(${h}, ${saturation}%, ${lightness}%, ${alpha})`;
 }
```

Comment:
The `stringToHSLAColor()` function can be removed since `getGraphColorPalette()` is replacing it.